front-end/src/components/Post/GovernanceSideBar/Proposals/ProposalVoteInfo.tsx

```diff
@@ -51,28 +56,44 @@ const ProposalVoteInfo = ({ className, proposalId }:  Props) => {
 	}, [api, apiReady, proposalId]);
 
 	return (
-		<Card className={className}>
-			<h3>Overview</h3>
-			<Grid columns={3} divided>
-				<Grid.Row>
-					<Grid.Column>
-						<h6>Deposit</h6>
-						<div>{deposit}</div>
-					</Grid.Column>
-					<Grid.Column>
-						<h6>Seconded by</h6>
-						{seconds ? <div>{seconds} addresses</div> : null}
-					</Grid.Column>
-					<Grid.Column>
-						<h6>Locked {chainProperties[currentNetwork].tokenSymbol}</h6>
-						<div>{seconds * parseInt(deposit.split(' ')[0]) || 0}</div>
-					</Grid.Column>
-				</Grid.Row>
-			</Grid>
+		<Card className={loadingStatus.isLoading ? `LoaderWrapper ${className}` : className}>
+			{loadingStatus.isLoading
+				?
+				<Loader text={loadingStatus.message}/>
+				: apiReady
```

Comment:
I think you should not rely on `apiReady`. The loading status should be set to `false` whenever the api is ready, and you can avoid this `apiReady` condition.

Reply:
Yes, I also wasn't a fan of the nested ternary here. I was just thinking that in case there're issues with the api, we would never show an error message to the user.

Reply:
This is a valid thing, but shouldn't be implemented here. If we want to make this better, we should have a timeout in the Loader component, and a `timeout` + `timeoutMessage` props. So that past `timeout` we don't show the Loader any more, but the  `timeoutMessage`

Reply:
Right, agree.